src/components/core/newsletter/index.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import React from "react";
+import { Linkedin, Facebook ,Twitter , Instagram  } from 'lucide-react';
+
+const Newsletter: React.FC = () => {
+    return <div className="w-full ">
+        <div className="flex flex-col justify-center items-center  gap-6 px-20 py-10 ">
+            <h4 className="text-xs uppercase">Newsletter</h4>
+            <h2  className="text-xs md:text-base uppercase text-primary-light text-center">Sign up for latest updates and offers</h2>
+            <div className="flex flex-col gap-4 md:flex-row">
```

Comment:
In light mode, it is unclear if an input is even present, put something like a border around it for light mode.